task_opts_unix.go

```diff
@@ -103,3 +103,55 @@ func WithShimCgroup(path string) NewTaskOpts {
 		return nil
 	}
 }
+
+// WithUIDMap allows console I/O to work with the remapped uid
+func WithUIDMap(uid uint32) NewTaskOpts {
```

Comment:
since a map isn't involved, for future contributors/maintainers who are trying to understand the code, I think this should simply be `WithUIDOwner` as it takes a UID and sets it as the owner for the IO opts struct.

Reply:
Renamed as suggested!